test/local/senders/email_service.js

```diff
@@ -103,7 +103,7 @@ describe(
             errno: 106,
             error: 'BounceComplaintError',
             message: 'FREAKOUT',
-            bounceAt: 1533641031755
```

Comment:
was `bounceAt` incorrect previously? Did the tests not catch it because `bouncedAt` wasn't tested for?

Reply:
Yep, it was a typo and there were no assertions around to pick up on it.